packages/components/src/color-palette/index.js

```diff
@@ -112,19 +112,27 @@ function MultiplePalettes( {
 	);
 }
 
-export function CustomColorPickerDropdown( { isRenderedInSidebar, ...props } ) {
+export function CustomColorPickerDropdown( {
+	isRenderedInSidebar,
```

Comment:
Not introduced here, but this is the first time I've seen it. `isRenderedInSidebar` doesn't seem like it should be in the components package. The prop is very specific to the editor's right-hand sidebar. What if there's a sidebar on the left of the screen?

It looks like it was a late fix for a WordPress release - https://github.com/WordPress/gutenberg/pull/37115. @jorgefilipecosta, as you introduced this would you be willing to revisit it?

My feeling is that it should be more generic, like a `childPopoverPlacement` prop that's passed through to any popovers that should cascade in a given direction.

Reply:
I agree 100% on this one, and coincidentally I have just opened an issue with a few changes that I'd like to see for `GradientPicker`:

- https://github.com/WordPress/gutenberg/issues/42994

> My feeling is that it should be more generic, like a `childPopoverPlacement` prop that's passed through to any popovers that should cascade in a given direction.

For the specific prop, it will largely depend on what we decide design-wise — let's see how this conversation evolves — but in general, I'd _love_ to get rid of that `isRenderedInSidebar` prop (also considering how many times it's forwarded through the various component chain!)

Reply:
I agree it would be good to get rid of isRenderedInSidebar it was added as a short fix for a visual issue.

Reply:
Hi @ciampo, I will try to find some availability to work on some enhancements to this component.

Reply:
That's great, thank you! I've assigned [the issue with the list of enhancements ](https://github.com/WordPress/gutenberg/issues/42994) — feel free to ping for any help and for reviews :)